examples/multitask-monorepo/vpc/main.tf

```diff
@@ -0,0 +1,32 @@
+module "vpc" {
+  source  = "registry.terraform.io/terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = "${var.env}-vpc"
+  cidr = "10.0.0.0/16"
+
+  azs = [
+    "us-east-1a",
```

Comment:
```suggestion
    "${var.aws_region}a",
```
And for other below

Reply:
I can see this method for the first time. Could I use the old one?

Reply:
I'd say use a variable.